automation/unit-tests.sh

```diff
@@ -6,26 +6,30 @@ make generate
 templates=`ls dist/templates/*`
 namespace="kubevirt"
 
-oc apply -f - <<EOF
```

Comment:
can you please keep creating of namespace here? Unit tests lane is failing due to this change

Reply:
Done. :+1: